grassroots-backend/src/App.module.ts

```diff
@@ -6,13 +6,51 @@ import { TypeOrmModule } from "@nestjs/typeorm";
 import { ConfigModule, ConfigService } from "@nestjs/config";
 import { ContactEntityOutDTO } from "./grassroots-shared/Contact.entity.dto";
 import { NestExpressApplication } from "@nestjs/platform-express";
+import { AuthModule } from "./auth/Auth.module";
+import { UsersModule } from "./users/Users.module";
+import { PassportModuleImport } from "./auth/PassportModuleImport";
+import { UsersService } from "./users/Users.service";
+import { UserEntity } from "./grassroots-shared/User.entity";
+
+// To get these imports working both inside and outside jest, we need to play some ugly tricks.
```

Comment:
I don't know enough about the various ways things can be imported to know if there's a better approach here. I tried a bunch of options, and this is what worked...